tasks/targets.py

```diff
@@ -176,12 +189,16 @@ def __init__(self, tag, task):
         self._tag = tag
         self._task = task
 
+    _tag_cache = {}
+
     def get(self, session):
         '''
         Return a copy of the underlying OBSColumn in the specified session.
```

Comment:
This should be `Return a copy of the underlying OBSTag in the specified session.`